winpr/libwinpr/sspi/test/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(${MODULE_PREFIX}_TESTS
 	TestInitializeSecurityContext.c
 	TestAcquireCredentialsHandle.c
 	TestCredSSP.c
-	TestSchannel.c
+	#TestSchannel.c
```

Comment:
Schannel isn't completely finished and the the test does the following
```
return 0; /* disable by default - causes crash */
```
Therefore I disabled it completely since I thought it doesn't make much sense to compile a faulty test.